app/views/devise/passwords/new.mobile.haml

```diff
@@ -18,10 +18,10 @@
               = f.text_field :email
 
           .controls
-            = f.submit t('devise.passwords.new.send_password_instructions'), :class => 'btn'
+            = f.submit t('devise.passwords.new.send_password_instructions'), :class => 'btn btn-primary'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.